### ISSUE TITLE
How long did it take for you to train the model ?

### ISSUE THREAD

**Issue by rakshit-ce** [hpanwar08/detectron2#35]
Hi, I'm currently using the pre-trained model weights supplied in the repo. But, I need to fine-tune the model further to suit my requirements. So, I ran a sample training with only one image and one instance/bbox in that image, but the training script says it needs 1 day 18 hrs to complete training on this minuscule training set. Is this an issue with ETA estimation or does the model really take that long to train on one image ??

**Comment by hpanwar08** [comment]
What's your training config?

**Comment by rakshit-ce** [comment]
I'm using your script `train_net_dla.py` for fine-tuning on my data. I've just changed few lines as you had suggested before !!

**Comment by hpanwar08** [comment]
Reduce the number of iterations in the relevant config https://github.com/hpanwar08/detectron2/blob/master/configs/DLA_mask_rcnn_R_101_FPN_3x.yaml#L14

**Comment by rakshit-ce** [comment]
How long did it take for you to train the models, and on which platform ?

**Comment by hpanwar08** [comment]
6days for each model on windows10. It is better to train on linux.

**Comment by rakshit-ce** [comment]
Which script to use for fine-tuning ? Also, how many iterations do you recommend for fine-tuning the weights on my custom-training set ? I've around 1000 images with 5-8 instances/bboxes per image

P.S: I'm currently using slightly modified `train_net_dla.py` script from `tools` folder

**Comment by hpanwar08** [comment]
You can use the same script for fine tuning with low learning rate. Try with different values for iteration. You can start with 5000 iteration assuming 1 image per batch.
